src/App.css

```diff
@@ -1,8 +1,63 @@
 .app {
   background-color: #0d0119;
   color: #ccc;
+  min-height: 100vh;
 }
 
 .app__hero-table tbody {
   vertical-align: top;
 }
+
+.counter-pick {
+  padding: 10px;
+}
+
+.title {
+  font-weight: bold;
+}
+
+.counter-pick .case-item {
```

Comment:
Since `CaseItem` is not an independent component (he exists only in terms of `CounterPick` component) it should use BEM ideology on CSS naming.
Something like `.counter-pick__case-item` and `.counter-pick__case-details`

See https://en.bem.info for details